### DISCUSSION TITLE
No proofs found since 1.2.0

### DISCUSSION THREAD

**Discussion by racergoodwin** [Chia-Network/chia-blockchain#7939] [Support]
Has anyone else noticed that they aren't finding any proofs (or winning any XCH), since updating to 1.2.0?
I am running two completely independent farms, different locations, with different internet connections. Both are run on Ubuntu 20.04 with all latest updates, and chia is updated to the latest version (1.2.3 at time of writing this).
Farm 1 has 3311 plots on 2 harvesters with ~1200 OG plots each and the full node server has over 900 plots and increasing, in self pool plots.
Farm 2 only has the full node server with 2930 plots. No harvesters. Total of 2344 OG plots and 586 self pool plots.
Logs for both farms (including harvesters) are set to INFO and seem to show no problems with anything.
Statistically, with 6241 plots the odds of winning are around 1 in 12 days, but approx 4 weeks on and not even any proofs found, I am wondering what is happening.

**Comment by max4711qq** [comment]
did you win any blocks since you‘ve posted? I am farming 3PiB for 17 days without a win. Time to Win is 2 days. Log files are OK. Response times 99% < 2s. 
I guess my dry period started also with updating to 1.2.x
Upgraded several times during the last 2 weeks. No change. Before that my wins were pretty much in line with the indicated Time to Win. I am about to stop farming and wiping clean my servers. So much frustration with Chia.

**Reply by racergoodwin** [reply]
No, I still have nothing.
No partial or complete proofs found.
I have done a clean install of Ubuntu 20.04 and Chia on a new server but kept the same plots, with no difference to outcome.
I have no invalid plots and the logs show all my timings are less than 1 second.
I was also getting a regular increase in XCH in line with my increase in number of plots as the blockchain grew when using 1.1.7 or before with OG solo plots.
My farm is now a mixture of Original and Pool plots.
What OS are you using?

**Reply by max4711qq** [reply]
I am on Ubuntu 20.04 as well. 

1 x Node/Farmer/Harvester with some local plots. Response time for the last 24 hours was 1.4s in average. Nothing longer than 2s.

3 x Remote Harvester with local Plots. Response times are <1s in average.

Total plots (all OG plots) now 3.33 PiB

No plotting on those machines. Servers are powerful and have plenty of free resources. (Dual XEON V4, 256GB RAM, SSD). Nothing else beside Chia on them.

Also checked for invalid plots. It looks all good to me. No errors. Clean log files, etc...

**Comment by alexwwstx** [comment]
I am a small finish with 3000 OG plots before, get nothing for almost 2 months. Started to replot NFT plots and joined a pool 3 weeks ago. Very frustrated to solo farming.
what plotter do you guys use? 50% of my 3000 OG was done by official chia plot and another 50% is by madmax. Now all NFTs are by Madmax

**Reply by max4711qq** [reply]
All my plots have been created using SwarChiaPlotManager

**Reply by racergoodwin** [reply]
90% of my OG plots were done with the official plotter, before madMAx43v3r was released. I then ran 2 farms side by side, one with the OG plots and another that only had plots created with madMAx to see if it worked. Once I'd won once with madMAx, I then switched all plots over to madMAx.

**Comment by LYXQQ** [comment]
I have the same problem and feel anxious.
8000 OG and 3000 NTFS(by Madmax)

**Comment by racergoodwin** [comment]
I have now setup a completely new farm and am only going to be creating solo plots, not enabling the NFT, to see what happens.

**Comment by 21497724** [comment]
Same issue, not easy to get reward after update to larger than 1.2.0

**Reply by racergoodwin** [reply]
You say 'not easy', does that mean you have won something since 1.2.0 upgrade?

**Reply by 21497724** [reply]
yes, I use to get 8 blocks/day before 1.2.0. Now 3~4/day

**Comment by kindofblue** [comment]
same here, > 7000 plots solo, win 2xch every two weeks, but never since 1.2.0.

**Comment by marssystems** [comment]
You are probably just unlucky.
I have 1037 OG plots and I solo farm.
Since upgrading to ver. 1.2.3, I have found 2 proofs - 4 XCH - result!!

**Reply by alexwwstx** [reply]
do you have a dedicated farmer w/o running any plotter? or also running plotter with your farmer?

**Comment by marssystems** [comment]
I was running a plotter at the same time but I've stopped now as I don't have any more disk space.

**Reply by racergoodwin** [reply]
To confirm, marssystems, you haven't started self pooling, yet? You are still only running solo plots?

**Comment by marssystems** [comment]
Correct - no self pooling. I'm still on the old-style  plots. I think its still to early to jump in to self pooling or to join a pool as the software is still "too new" to risk it. I will wait till it settles down a bit before I change anything on my system. At the moment I'm going with what works.

**Reply by alexwwstx** [reply]
How many OG do you totally have since you updated to 1.2.3? 
That's very lucky to get 2 blocks by 1037 OG only.

**Comment by marssystems** [comment]
I had about 650 OG plots before I upgraded to 1.2.3 and I just carried on plotting until I ran out of disk space last week. So now I have 1037 plots and I will stick with them as I'm not planning to invest in any more hard drives until I recover the costs of my current system.

And - YES - you're right - I think I am very lucky to get 2 proofs in such a short space of time and 4 proofs since I started about 10 weeks ago.

**Reply by alexwwstx** [reply]
Wow, I will say super lucky for your case. 
What plotter did you use, chia official GUI, CLI, Madmax?

**Comment by marssystems** [comment]
As I'm new to this Chia game - I just used the plotter that is in the GUI.
I didn't know about anything else at the time - and I still don't know that much now - but learning all the time.

**Comment by racergoodwin** [comment]
Still trying to find commonality amongst those experiencing this same issue.
@max4711qq @alexwwstx @LYXQQ @21497724 @kindofblue 
What OS are you using?
How did you all start the NFT? Did you use command line or GUI?

**Reply by alexwwstx** [reply]
My OS is Ubuntu 21.04, and I am using CLI only, switched to Madmax since middle June

**Reply by racergoodwin** [reply]
OK, so our only obvious similarity is Ubuntu, though different versions.

**Reply by racergoodwin** [reply]
Did you follow the upgrade process from 1.1.7 through each version to 1.2.3 or did you at any time do a completely clean install and then import your keys again?

**Reply by alexwwstx** [reply]
strictly follow the steps in Git like the following

```
cd chia-blockchain
. ./activate
chia stop -d all
deactivate
git fetch
git checkout latest
git reset --hard FETCH_HEAD --recurse-submodules

# If you get RELEASE.dev0 then delete the package-lock.json in chia-blockchain-gui and install.sh again

git status

# git status should say "nothing to commit, working tree clean", 
# if you have uncommitted changes, RELEASE.dev0 will be reported.

sh install.sh

. ./activate

chia init
```

**Reply by lazydrone** [reply]
I have witnessed the same and don't know if there's any correlation or just a string of bad luck, but I haven't won a block since upgrading to 1.2.X.  Currently I'm right at 300% luck (27 days since win with a time to win of 9 days (8000 OG Plots)  
I'm using Windows.  (1) Farmer/Harvester  (3) Plotters.  I've recently starting harvesting from my plotters but I physically move the drive from the plotter to the farmer/harvester.  I started noticing some latency issues on my main farmer/harvester with 50 drives attached so on another plotter I'm consolidating additional drives there.  The remote harvesters seem to pass filters faster than the main farmer/harvester which is a little odd.

Have run Chia Plots Check on all the plots for the last month or so.  handful of bad plots but 99%+ are valid.  Used SWAR in the beginning but primarily using madmax (windows version) now.

**Comment by marssystems** [comment]
I am using Windows 10.

**Reply by racergoodwin** [reply]
Good to know. Thanks.

**Comment by LYXQQ** [comment]
Win10, OG plots use command line,NFT plots use Madmax.
Whether NFT and og plots cannot be farming with the same harvester?

**Reply by alexwwstx** [reply]
seems really lucky or unlucky now
NFT and OG can be mined together

**Reply by max4711qq** [reply]
“20 days no wins with time to win = 2 days” is not just unlucky I think

**Reply by alexwwstx** [reply]
did you try to git switch back to your previous 'win' version?

**Reply by max4711qq** [reply]
no. I've never downgraded the system ever.

**Reply by alexwwstx** [reply]
if I were you, definitely that's what I would try as the ETW is only 2 days, so you should see if you can get blocks during a week

**Comment by Jacek-ghub** [comment]
@racergoodwin @max4711qq 

Maybe you should bring it up on the [Issues ](https://github.com/Chia-Network/chia-blockchain/issues?page=1&q=is%3Aopen+is%3Aissue)page?  

Looks like you have zero support from the team chia here.  In that part, there is potentially more people to support your case.  

I have a smaller dataset (on Windows), but am also way past the average time to win.  In my case, it could be a bad luck, but too many people with bad luck may rather mean a bug in the software.

**Comment by racergoodwin** [comment]
As of this morning, one of my harvesters that has only got OG plots and is running version 1.2.3 on Ubuntu 20.04 "Found 1 plot" and my wallet received the 0.25 and 1.75 XCH.

**Reply by max4711qq** [reply]
I've also received 2 XCH yesterday with mit 3 PiB Setup. (After more than 20 days with ETW = 2 days)
But I think this even makes the whole thing worse: I guess it's not a local bug/broken setup or "just a matter of luck". I've looked at this for a while now and talked to some other guys who are farming >1PiB setups. Nobody is in line at all with ETW anymore for the last 3-4 weeks. Good Luck - hopefully you will get more blocks now.

**Comment by marssystems** [comment]
Congratulations! - I told you it was just a matter of luck.

**Comment by Jacek-ghub** [comment]
Take a look at [this](https://www.youtube.com/watch?v=_yEBS-q0sK4) (semi-official??) video as it explains when you should start worrying about not winning.  With petabyte size farms it is just math, not really "a matter of luck."

**Comment by alexwwstx** [comment]
I finally got one block hit today....

**Comment by Jacek-ghub** [comment]
Question to all of you who just recently got your blocks.  Have you been continuously adding plots?

I am running PSChiaPlotter heatmaps against my harvesters/plotters, and it looks like the plots that I generated recently may be passing more filters than the ones generated maybe even a couple of weeks ago.  Also, old NFT plots looks like are finding less partials than the recent ones.  I cannot say for sure that this is the case, but it kind of looks like that to me.

I still have old OG plots generated with the official chia plotter, but am using MadMax for quite long now.  Maybe things got improved with MM v1.1.5 (for Windows).  However, that is just recompile of the original sources, so I don't think that it is Windows related (if MM related at all).

Also, I would rather not bet on the problems with MM, but rather how it is being asked to generate those plots (seeds that are given to generate the plot).

**Reply by max4711qq** [reply]
I am getting pretty much exactly "1 filter passed" out of 512 attempts. (This is as it should be - at least according to my understanding)
This is also my long term average and this never changed. I am farming with almost 40k plots. (All OG, produced between May and now. Still adding more plots daily)

**Reply by Jacek-ghub** [reply]
How do you get your the 1:512 ratio?  Is it somewhere in the logs?

Not sure if you tried to use that Excel doc.  When you plug there 40k plots and just 30 days ,you should be getting about 10 blocks with that time period (most likely).  Also, if you didn't get anything for 6 days in a row, you should be seriously concerned.    At least, that is how I understand that spreadsheet.

As we are all trying to find commonality, it doesn't look like that is either obvious, or rather farm specific.  It looks more like when the farm was created, and was seeded with some random seed, that was the actual source.  Or, as I tried to say, maybe our plotters were somehow unlucky in getting seeds to generate our plots.  That would be IMO one explanation that things just improved to actually several people within a day or two.  

Although, it is really sad that no official Chia guy bothered to chime in.

Seeing all that, I would hope that the next release would have some better analytics that would show the health of the farm.  Right now, the UI is just garbage.

**Reply by max4711qq** [reply]
yes, it's in the logs. But you have to do some maths on your own to calculate this. Also "Chiadog" can help you to get those details about "plots passing filter" and how many "signage points/challenges" you've participated in. (Much easier to do this with Chiadog!)
But I am pretty sure that this is ca. (!) 1 out of 512 for everyone. No matter how or when the plot was created, as long as it's a valid plot. 

I don't need an Excel doc to understand that I am way beyond statistical possibilities of "luck". Therefore I've written in this thread about it, when it was created:
https://github.com/Chia-Network/chia-blockchain/discussions/7939#discussioncomment-1140359

Update:
I've won 1 (ONE) block in about 25 days. (>3 PiB and time to win 2 days. Logs are OK. System is always up and running)

Therefore it's clear that something is seriously wrong. As I've stated before, I also know some other "farmers" with >1 PiB with the same problem.
At least we see that it's not working and we do have a result/experience on hand that tells us to stop this better sooner than later. Unfortunately there are so many small farmers out there who will not get this understanding anytime soon. With a time to win of 2 months, it might take you 2 years to understand that you are not getting what you are promised.

I'll watch this for a few more days - maybe until a new release - and then I'll finally wipe the systems clean. Cheers.

**Reply by Jacek-ghub** [reply]
I understand your frustration, as I also face the same problem.

However, trying to generalize based on that is possibly wrong.  One thing that speaks against that is that all pools (whether OG or NFT) looks like are not hit with that problem (and some have big farms there), so there has to be an issue related to our setups (how the farm was created, or how plots are created - but not related to anything on client side, but rather on the chia network side - ignoring our farms).

I didn't have much activity on either my OG plots or NFTs using self-pooling for over two months.  Therefore, I started replotting. and joined a pool  Things didn't initially improve, but again those heatmaps are showing more activity in the last week or so, and that strangely correlates with those recent wins of few people in this thread.  Again, this is just correlation, or rather I just want to see it this way.

**Reply by max4711qq** [reply]
I am not generalizing this for the whole network. Just saying I know some ! guys being in the very same situation: Clean setups, wins way beyond statistical possibilities. And I am worried that if this issue happens to smaller “farmers”, they might only find out when it’s too late... 
Anyway. Good luck

**Reply by alexwwstx** [reply]
I did replot to NFT plots since middle July.

**Reply by alexwwstx** [reply]
for any farm > 1P, the invested $ should be > 20k on HDDs w/o considering other cost. Not sure how to get money back and how to handle those HDDs if give up Chia w/o figuring out what's going on.

**Reply by rckjhn** [reply]
Same here, ~15000 plots was winning regularly until mid July. Since then nothing. Ubuntu 20.04.

**Reply by rckjhn** [reply]
It is entirely possible that out of ~35 EiB that we are just the very small population of farmers with bad luck but it would be nice to have some official tools to verify farm health.

**Reply by racergoodwin** [reply]
My second farm has also won a block now. Still a little behind the predicted time to win, so maybe just was an unlucky streak. Only time will tell.
I have been continually adding plots and started a 3rd farm without NFT plots, to see if that was more lucky.

**Reply by Jacek-ghub** [reply]
I think that we all may be barking against the wrong tree.  It looks to me that the problem is not really farm or plotter related but rather the interaction between plots and the chia network server side (how the server side decides which block wins).  

If this the issue would be farm related, we would start seeing some patterns, but we see zilch so far.  I am constantly running **PSChiaPlotter** heatmaps against my harvesters, also thanks to @max4711qq started running **ChiaDog**, and am more convinced that my old plots (both OGs and NFTs) are doing nothing, while those NFTs plotted within the last week+ are several times more active.  Maybe those who won those blocks were still adding new plots, and that is thanks to those.

Maybe those of you that won blocks recently could check the dates on the winning blocks.  Maybe that is the pattern.

**Comment by BrandtH22** [comment]
@racergoodwin , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
